front/app/containers/Admin/projects/project/nativeSurvey/FormResults/FormResultsQuestion/TextQuestion/Analysis/AnalysisInsights.tsx

```diff
@@ -106,6 +106,10 @@ const Summary = ({
     task?.data.attributes.state === 'queued' ||
     task?.data.attributes.state === 'in_progress';
 
+  const refreshDisabled =
+    missingInputsCount === 0 ||
+    (largeSummariesEnabled && filteredInputCount + missingInputsCount > 30);
```

Comment:
Technically, I think it could be possible that some inputs were removed? In that case I think `filteredInputCount + missingInputsCount` could be larger than 30 while actually the total amount of inputs is less than 30?

Reply:
If some inputs were indeed deleted, why would that not be reflected in the filteredInputCount / missingInputCount respectively?

Reply:
I'm not sure if `filteredInputCount` is the new count of inputs (in which case I don't think we should add `missingInputCount`), or if it's the count of inputs when it was generated the last time (in which case inputs could have been removed). The missing inputs count only counts inputs that were added since the last time it was generated and ignores inputs that were removed since the last time it was generated.